x/ecocredit/server/keeper.go

```diff
@@ -0,0 +1,14 @@
+package server
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// In this keeper.go files we expose methods the basket.Keeper needs.
+
+func (s serverImpl) GetCreateBasketFee(ctx context.Context) sdk.Coins {
+	//TODO implement me
+	panic("implement me")
+}
```

Comment:
Should we add this as a separate item in #732 or is the idea to include it with `CreateBasket`?

Reply:
the idea is to include with `CreateBasket` but just stub out an example